frameworks/bf/run-bf.js

```diff
@@ -0,0 +1,7 @@
+var execSync = require('child_process').execSync;
+module.exports.runBf = function(input) {
```

Comment:
Thanks, I'm now finally getting a non-empty STDOUT.  It appears that the `bf` fast Brainf**k Interpreter doesn't just output the expected string though - it outputs a buffer instead with hexadecimals - but that shouldn't be too hard to fix, I think (maybe `5/6 kyu` in Codewars terms :wink:)